cmd/lassie/main.go

```diff
@@ -75,8 +75,11 @@ func before(cctx *cli.Context) error {
 		level = "DEBUG"
 	}
 
```

Comment:
I'd like to keep the setting of the log level across subsystem even if an environment variable is provided.

```suggestion
        if os.Getenv("GOLOG_LOG_LEVEL") != "" {
                level = os.Getenv("GOLOG_LOG_LEVEL")
        }

        for _, name := range subsystems {
	        _ = logging.SetLogLevel(name, level)
        }
```

Reply:
`GOLOG_LOG_LEVEL` can be somewhat more complex / have semantics for subsystems: https://github.com/ipfs/go-log

e.g `GOLOG_LOG_LEVEL="error,lassie=info,lassie/bitswap=debug"` will be interpreted as you might hope

Reply:
OH. That's neat. So then when GOLOG_LOG_LEVEL is specified, it's overriding all of our subsystem log level defaults as well as flag specified logging levels (`--verbose` and `--very-verbose`). Yeah, that makes sense to me.